libraries/integration/Microsoft.Bot.Builder.Integration.ApplicationInsights.WebApi/Microsoft.Bot.Builder.Integration.ApplicationInsights.WebApi.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0"?>
+<doc>
+    <assembly>
+        <name>Microsoft.Bot.Builder.Integration.ApplicationInsights.WebApi</name>
+    </assembly>
+    <members>
+        <member name="T:Microsoft.Bot.Builder.Integration.ApplicationInsights.WebApi.TelemetryBotIdInitializer">
+            <summary>
+            Initializer that sets the user ID based on Bot data.
```

Comment:
>ets the user I [](start = 30, length = 14)

And session ID (in addition to other bot-specific properties such as activityId)

Reply:
Updated in Microsoft.Bot.Builder.Integration.ApplicationInsights.WebApi\TelemetryBotIdInitializer.cs